mvnw test fails to detect some test failures

### Description

Maven surefire plugin version 3.0.0-M6 fails to detect some importer test failures, probably the same issue reported here: https://issues.apache.org/jira/projects/SUREFIRE/issues/SUREFIRE-2079

As a result, our importer test github workflow false positively report success for the recent PRs. Downgrading to surefire 3.0.0-M5 fixes the problem.

### Steps to reproduce

Run the following command:

```bash
# ./mvnw test "-Dtest=com.hedera.mirror.common/**/*.java,EntityRecordItemListenerScheduleTest#scheduleCreateLongTermScheduledTransaction" -pl hedera-mirror-importer
```

output with surefire 3.0.0-M6

```
2022-05-23T22:16:03.714-0600 INFO main c.h.m.i.p.r.e.s.SqlEntityListener Completed batch inserts in 289.2 ms
[ERROR] :$�:java.lang.AssertionError:
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0, Time elapsed: 25.544 s - in com.hedera.mirror.importer.parser.record.entity.EntityRecordItemListenerScheduleTest
[INFO]
[INFO] Results:
[INFO]
[INFO] Tests run: 0, Failures: 0, Errors: 0, Skipped: 0
[INFO]
[WARNING] Corrupted channel by directly writing to native stream in forked JVM 1. See FAQ web page and the dump file /Users/xin/work/hedera-mirror-node/hedera-mirror-importer/target/surefire-reports/2022-05-23T23-15-29_539-jvmRun1.dumpstream
```

output with surefire 3.0.0-M5

```
2022-05-23T22:15:02.673-0600 INFO main c.h.m.i.p.r.e.s.SqlEntityListener Completed batch inserts in 316.4 ms
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0, Time elapsed: 24.611 s <<< FAILURE! - in com.hedera.mirror.importer.parser.record.entity.EntityRecordItemListenerScheduleTest
[ERROR] com.hedera.mirror.importer.parser.record.entity.EntityRecordItemListenerScheduleTest.scheduleCreateLongTermScheduledTransaction  Time elapsed: 1.469 s  <<< FAILURE!
java.lang.AssertionError:

Expecting ArrayList:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=0, receiverSigRequired=null)]
to contain only:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=null, receiverSigRequired=null)]
element(s) not found:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=null, receiverSigRequired=null)]
and element(s) not expected:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=0, receiverSigRequired=null)]

	at com.hedera.mirror.importer.parser.record.entity.EntityRecordItemListenerScheduleTest.scheduleCreateLongTermScheduledTransaction(EntityRecordItemListenerScheduleTest.java:170)

[INFO]
[INFO] Results:
[INFO]
[ERROR] Failures:
[ERROR]   EntityRecordItemListenerScheduleTest.scheduleCreateLongTermScheduledTransaction:170
Expecting ArrayList:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=0, receiverSigRequired=null)]
to contain only:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=null, receiverSigRequired=null)]
element(s) not found:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d0
174                 <!-- antlr 2.x (Hibernate dependency) is so old it doesn't have sources in Maven Central -->
04042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=null, receiverSigRequired=null)]
and element(s) not expected:
  [Entity(super=AbstractEntity(autoRenewAccountId=null, autoRenewPeriod=null, createdTimestamp=1653366708404351000, deleted=false, expirationTimestamp=null, id=2, maxAutomaticTokenAssociations=null, memo=cLAf7VIDVpy3qdnd, num=2, proxyAccountId=null, publicKey=804cbb58260c253d0fcf959ace6f6686ab7b0bfcf9affdb6a318d004042f89fd, realm=0, shard=0, type=SCHEDULE, timestampRange=[1653366708404351000..+∞)), ethereumNonce=0, receiverSigRequired=null)]

[INFO]
[ERROR] Tests run: 1, Failures: 1, Errors: 0, Skipped: 0
```

### Additional context

_No response_

### Hedera network

other

### Version

v0.57.0

### Operating system

_No response_

--- a/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/batch/BatchUpserterTest.java
+++ b/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/batch/BatchUpserterTest.java
@@ -756,6 +756,7 @@ class BatchUpserterTest extends IntegrationTest {
         Entity entity = new Entity();
         entity.setId(id);
         entity.setCreatedTimestamp(createdTimestamp);
+        entity.setEthereumNonce(0L);
         entity.setTimestampLower(modifiedTimestamp);
         entity.setNum(id);
         entity.setRealm(0L);

--- a/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/AbstractEntityRecordItemListenerTest.java
+++ b/hedera-mirror-importer/src/test/java/com/hedera/mirror/importer/parser/record/entity/AbstractEntityRecordItemListenerTest.java
@@ -295,7 +295,7 @@ public abstract class AbstractEntityRecordItemListenerTest extends IntegrationTe
 
     protected TransactionRecord buildTransactionRecordWithNoTransactions(
             Consumer<TransactionRecord.Builder> customBuilder, TransactionBody transactionBody, int status
-    ){
+    ) {
         TransactionRecord.Builder recordBuilder = TransactionRecord.newBuilder();
         recordBuilder.setConsensusTimestamp(Utility.instantToTimestamp(Instant.now()));
         recordBuilder.setMemoBytes(ByteString.copyFromUtf8(transactionBody.getMemo()));
@@ -397,6 +397,7 @@ public abstract class AbstractEntityRecordItemListenerTest extends IntegrationTe
         entity.setAutoRenewPeriod(autoRenewPeriod);
         entity.setCreatedTimestamp(createdTimestamp);
         entity.setDeleted(deleted);
+        entity.setEthereumNonce(0L);
         entity.setExpirationTimestamp(expiryTimeNs);
         entity.setMemo(memo);
         entity.setTimestampLower(modifiedTimestamp);

--- a/pom.xml
+++ b/pom.xml
@@ -205,7 +205,7 @@
                 <plugin>
                     <groupId>org.apache.maven.plugins</groupId>
                     <artifactId>maven-surefire-plugin</artifactId>
-                    <version>3.0.0-M6</version>
+                    <version>3.0.0-M5</version>
                     <configuration>
                         <!-- Jacoco prepare-agent builds some command-line params without -->
                         <!-- which jacoco will not instrument. Hence it is important to add -->

